lib/configproxy.js

```diff
@@ -337,6 +340,7 @@ ConfigurableProxy.prototype.handle_proxy = function (kind, req, res) {
         this.handle_proxy_error(404, kind, req, res);
         return;
     }
+    this.emit("proxy_request", req, res);
```

Comment:
That's true, but it's also how the whole proxy works - listening of NHP events to set up the proxying. Is there anything we should be doing differently? try/catch?